app/webpack.config.js

```diff
@@ -5,103 +5,105 @@ var InlineEnviromentVariablesPlugin = require('inline-environment-variables-webp
 var path = require('path');
 
 var env = new webpack.DefinePlugin({
-    'process.env': JSON.stringify(process.env),
+		'process.env': JSON.stringify(process.env),
 })
 
 var plugins = [
-    new ExtractTextPlugin('./public/css/styles.css', {
-        allChunks: true
-    }),
-    new InlineEnviromentVariablesPlugin([
-        'NODE_ENV',
-        'JWT_SECRET',
-        'DB_USERNAME',
-        'DB_HOST',
-        'DB_PASSWORD',
-        'DB_PORT',
-        'MAPBOX_ACCESS_TOKEN',
-        'S3_KEY',
-        'S3_SECRET',
-        'S3_BUCKET',
-        'STREAM_APP_ID',
-        'STREAM_KEY',
-        'STREAM_SECRET',
-        'ALGOLIA_APP_ID',
-        'ALGOLIA_SEARCH_ONLY_KEY',
-        'ALGOLIA_API_KEY',
-        'KEEN_PROJECT_ID',
-        'KEEN_WRITE_KEY',
-        'KEEN_READ_KEY',
-        'IMGIX_BASE_URL',
-        'API_URL'
-    ]),
+		new ExtractTextPlugin({
```

Comment:
Is the change to indentation with tabs in this file with a reason? If not can you change it back?